tool/presubmit.sh

```diff
@@ -14,7 +14,7 @@ echo "PASSED"
 
 # Make sure we pass the analyzer
 echo "Checking dartanalyzer..."
-FAILS_ANALYZER="$(find lib test -name "*.dart" | xargs dartanalyzer --options analysis_options.yaml)"
+FAILS_ANALYZER="$(find lib test -name "*.dart" | xargs dartanalyzer --options .analysis_options)"
```

Comment:
I'd prefer `.yaml` because `.files` are hidden by file managers and `ls` by default, and this file is just a regular hand-written project file.